### PR TITLE
My awesome phd work

### PR DESCRIPTION
**Pull request by tayloris**
I finished my Ph.D. last year at University of Bergen and here are my contributions using deal ii.

My latest paper "MB_paper_C" is finally published .

### CODE REVIEW

File: publications-2019.bib [hunk 1]
```diff
@@ -1035,4 +1035,26 @@ @Article{clevenger_stokes19
   Url                      = {https://arxiv.org/abs/1907.06696}
 }
 
+@Article{MB_paper_D,
+  author    = {Manuel Borregales and Kundan Kumar and Florin Adrian Radu and Carmen Rodrigo and Francisco Jos{é} Gaspar},
```
Comment (masterleinad) [inline publications-2019.bib:1039]:
```suggestion
  author    = {Manuel Borregales and Kundan Kumar and Florin Adrian Radu and Carmen Rodrigo and Francisco Jos\'e Gaspar},
```

File: publications-2019.bib [hunk 2]
```diff
@@ -1035,4 +1035,26 @@ @Article{clevenger_stokes19
...
+  year      = {2019},
+  volume    = {77},
+  number    = {6},
+  pages     = {1466–1478},
+  doi       = {10.1016/j.camwa.2018.09.005},
+  url       = {https://www.sciencedirect.com/science/article/pii/S0898122118305091},
+}
+
+@PhdThesis{MB_PhD_Thesis,
+  author    = {Manuel Antonio Borregales Rever{ó}n},
```
Comment (masterleinad) [inline publications-2019.bib:1051]:
```suggestion
  author    = {Manuel Antonio Borregales Rever\'on},
```

**Comment by tayloris** [comment]
Thanks for the suggestions, it is ready now.

**Pull request merged**